README.md

```diff
@@ -1,2 +1,102 @@
-# product-landing-page
-###this is a..
+# landing Page🚀
+
+## Project Description 📝
+
+> this project is about making landing page. it has showed HTML coding language, is uesful for web development students, included some specific information about HTML.
+
+
+- working with HTML flie inspired me to create this page.
+- this will solve both theoreical and practical problems in the level of intermediate.
+- during workung on this page, i learned more ideas about how creat Landing Page through HTML and CSS languages ,which codes i can use, how organize information together.
+- using of HTML and CSS languages make this Landing Page stand out.
+
+```html
+    <header id="header">
+      <div class="logo">
+        <img id ="header-img" src="./assets/logo.jpg" alt="logo">
+        <div class="btn">
+          <i class="fa fa-bars menu-btn">Travel</i>
+        </div>
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#features">Home</a></li>
+          <li><a class="nav-link" href="#how you can have safe travel">About</a></li>
+          <li><a class="nav-link" href="#which facilities provide to you">Services</a></li>
+          <li><a class="nav-link" href="#prices">Contact</a></li>
+        </ul>
+      </nav>
+      </div>
+    </header>
+```css
+*{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "poppins";
+}
+
+body {
+  background-image: url(./assets/river.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding-top: 150px;
+}
+
+
+
```

Comment:
- This is too long in here you just share a short peace of code that you really proud of a feature or something that you think is very rare in your code.
  - As this project only has HTML and CSS it is not required to add this part so you can remove it.